04.ls/ls.rb

```diff
@@ -33,4 +33,113 @@ def split_files(files)
   divided_files
 end
 
+PERMISSION = {
+  '0' => '---',
+  '1' => '--x',
+  '2' => '-w-',
+  '3' => '-wx',
+  '4' => 'r--',
+  '5' => 'r-x',
+  '6' => 'rw-',
+  '7' => 'rwx',
+  'fifo' => 'p',
+  'characterSpecial' => 'c',
+  'directory' => 'd',
+  'blockSpecial' => 'b',
+  'file' => '-',
+  'link' => 'l',
+  'socket' => 's'
+}.freeze
+
+def load_l_option(path, file_names)
+  stat_files = file_names.map { |faile_name| File.lstat("#{path}/#{faile_name}") }
```

Comment:
```suggestion
  stat_files = file_names.map { |file_name| File.lstat("#{path}/#{file_name}") }
```

ですね。

spell checkのプラグインとかを入れた方がいいかも？
[Code Spell Checker でスペルミスをチェックする \[VSCode\] \- もとひろ blog](https://motohiro-mm.hatenablog.com/entry/2023/10/11/131354)

Reply:
入れさせていただきました〜🙏